### PR TITLE
A set of challenges for chapters 1-7 of Vue 2 Tutorial Ep 1

### PR DESCRIPTION
**Pull request by shadowc**

### CODE REVIEW

File: challenges.yml [hunk 1]
```diff
@@ -36,3 +37,274 @@ extending_instancedynamicdata:
...
+            },
+          };
+          </script>
+          ```
+      correct: d
+
+extending_stylngacomponent:
+  question: |
+    What is the *minimum* code you need in a `<style>` tag (inside a `.vue`
+    single file component) to make it modular?
```
Comment (weaverryan) [inline challenges.yml]: I think we don't cover modular CSS until chapter 10.

**Review by weaverryan — COMMENTED** [review]
These are fun! At first glance, I don't really see any obvious, missing "concepts" that we should ask about during chapters 1-7

File: challenges.yml [hunk 2]
```diff
@@ -40,16 +40,15 @@ extending_instancedynamicdata:
...
   question: |
-    Imagine our application has a `.vue` single file component that renders as
-    the page title. It has a `title` data member that it uses to render the
+    Imagine you want to build a `.vue` single file component that renders a
+    page header. It needs to have a `title` data key that it uses to render the
     template.
 
-    Which one of these code snippets would compile correctly into such
-    component?
+    Which one of these code snippets would accomplish this?
```
Comment (weaverryan) [inline challenges.yml:47]: compile! Are you trying to scare PHP devs? ;) I softened this to sound less technical

Comment (shadowc) [reply challenges.yml:47]: LOL

**Pull request merged**